src/Common/Doctrine/Entity/Meta.php

```diff
@@ -167,6 +168,12 @@ public function update(
         $this->seoIndex = $seoIndex;
     }
 
+    public function __clone()
+    {
+        $this->id = null;
+        $this->url = Model::addNumber($this->url);
```

Comment:
hmm we can't be sure that this url will be unique
if you have my-url and my-url-2 and you clone my-url that will also turn in to my-url-2